src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.FruitTransaction.Operation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
```

Comment:
Basically, tests are written for services and repositories, it is not necessary to do this for models